org-eclipse-che-jdt-ls-extension/jdt.ls.extension.api/src/main/java/org/eclipse/che/jdt/ls/extension/api/dto/navigation/ImplementationsDescriptor.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2012-2017 Red Hat, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.jdt.ls.extension.api.dto.navigation;
+
+import java.util.List;
+import org.eclipse.lsp4j.SymbolInformation;
+
+public class ImplementationsDescriptor {
```

Comment:
If feel "ImplementersResponse" would be more precise. "Descriptors" can be all kinds of things. Also, we call the request parameter class "Parameter", so I think "Response" would fit better.